src/OpenTelemetry.Instrumentation.Dependencies/Implementation/HttpWebRequestActivitySource.net461.cs

```diff
@@ -13,7 +13,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 // </copyright>
-#if NET461
+#if NET452 || NET461
```

Comment:
Should we define a const like NETFRAMEWORK or NETFULL which will be set for NET452 NET461 and anything like NET47 etc in future?

Reply:
Well, NETFRAMEWORK is already a constant. I can change in those places. What do you think?

Reply:
NETFRAMEWORK target sounds good to me.